tests/Request/CityByUuidTest.php

```diff
@@ -106,6 +106,27 @@ public function testExecuteRealRequest($form, $instanceof)
     }
 
     /**
+     * @covers ::execute
+     *
+     * @dataProvider providerTestExecuteInvalidRequest
+     *
+     * @expectedException \InvalidArgumentException
+     *
+     * @depends      testExecute
+     */
+    public function testExecuteInvalidRequest($uuid, $language_codes)
+    {
+        $this->sut = CityByUuid::create($this->requestHandler);
+
+        $this->sut
+          ->setUuid($uuid)
+          ->setLanguageCodes($language_codes)
+          ->execute();
+
+        $this->fail('Invalid arguments must throw an exception.');
```

Comment:
This is redundant if `@expectedException` is specified. PHPUnit will log a test failure automatically if no exception of the specified type is thrown during the test.